stories/troubleshooting.stories.mdx

```diff
@@ -0,0 +1,59 @@
+<Meta title="Documentation/Troubleshooting"/>
+
+# Troubleshooting
+> Looking to get involved in the mono repo but running into issues? Read on to find out how to fix them!
```

Comment:
Just thinking, maybe using paragraphs rather than blockquotes might make the text a bit clearer? Semantically, blockquotes are for sections which are quoted from another source.

Reply:
Sure will take them out; just felt they helped readability in a wordy page